packages/babel-traverse/src/path/evaluation.ts

```diff
@@ -3,11 +3,38 @@ import type * as t from "@babel/types";
 
 // This file contains Babels metainterpreter that can evaluate static code.
 
-const VALID_CALLEES = ["String", "Number", "Math"] as const;
+const VALID_OBJECT_CALLEES = ["Number", "String", "Math"] as const;
+const NODE_MAJOR_VERSION = process.versions.node
+  ? parseInt(process.versions.node.split(".")[0])
+  : null;
+const VALID_IDENTIFIER_CALLEES = [
+  "isFinite",
+  "isNaN",
+  "parseFloat",
+  "parseInt",
+  "decodeURI",
+  "decodeURIComponent",
+  "encodeURI",
+  "encodeURIComponent",
+  NODE_MAJOR_VERSION >= 16 ? "btoa" : null,
+  NODE_MAJOR_VERSION >= 16 ? "atob" : null,
```

Comment:
```suggestion
  process.env.BABEL_8_BREAKING ? "btoa" : null,
  process.env.BABEL_8_BREAKING ? "atob" : null,
```
The `btoa` and `atob` support should be removed for Babel 7, because given the exact config, running Babel on Node.js 16 should produce output as same as the one generated on Node.js 14.

In Babel 8 we will drop Node 14 support, so we can add `btoa` and `atob` support then.

Reply:
All right, I didn't know it. Now I should have updated everything.